manageiq-operator/pkg/helpers/miq-components/httpd.go

```diff
@@ -175,6 +178,23 @@ func addOIDCEnv(secretName string, podSpec *corev1.PodSpec) {
 	podSpec.Containers[0].Env = append(podSpec.Containers[0].Env, clientSecret)
 }
 
+func getHttpdAuthConfigVersion(client client.Client, namespace string, spec *miqv1alpha1.ManageIQSpec) string {
+	httpd_auth_config_version := ""
+	if spec.HttpdAuthConfig != "" {
+		secret := &corev1.Secret{}
+		secretErr := client.Get(context.TODO(), types.NamespacedName{Namespace: namespace, Name: spec.HttpdAuthConfig}, secret)
+		if secretErr == nil {
+			httpd_auth_config_version = string(secret.GetObjectMeta().GetResourceVersion())
+		}
+	}
+	return httpd_auth_config_version
+}
+
+func setManagedHttpdCfgVersion(httpdAuthConfigVersion string, podSpec *corev1.PodSpec) {
+	managedHttpdCfgRevision := corev1.EnvVar{Name: "MANAGED_HTTPD_CFG_VERSION", Value: httpdAuthConfigVersion}
+	podSpec.Containers[0].Env = append(podSpec.Containers[0].Env, managedHttpdCfgRevision)
```

Comment:
Should we add a comment like `// This is not used by the pod, only to trigger a redeployment if the secret has changed`?